src/main/scala/RemoraApp.scala

```diff
@@ -54,10 +56,14 @@ object RemoraApp extends App with nl.grons.metrics.scala.DefaultInstrumented wit
     logger.info("Reporting metricsRegistry to Cloudwatch")
     val amazonCloudWatchAsync: AmazonCloudWatchAsync = AmazonCloudWatchAsyncClientBuilder.defaultClient
 
+    // if null, cloudwatch grabs all the logs possible
```

Comment:
Can you remove the comment, I think the metricFilter code is easy enough to understand that it doesn't need an additional comment to explain it here